src/App.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div id="app">
+    <CrucibleFilter />
     <CrucibleSearch />
     <CrucibleFilter />
```

Comment:
There are two `<CrucibleFilter />`. Is this a typo?

Reply:
fixed